Source/Charts/Components/YAxis.swift

```diff
@@ -175,5 +181,7 @@ open class YAxis: AxisBase
     @objc open var isDrawBottomYLabelEntryEnabled: Bool { return drawBottomYLabelEntryEnabled }
     
     @objc open var isDrawTopYLabelEntryEnabled: Bool { return drawTopYLabelEntryEnabled }
+    
+    @objc open var isLimitLineFillEnabled: Bool { return limitLineFill }
```

Comment:
Please remove this and rename the backing var to `isLimitLineFillEnabled`. I know this contradicts other boolean values in the library, but this is what they are all moving towards.

Reply:
done